models/news-models.js

```diff
@@ -0,0 +1,37 @@
+const db = require("../db/connection");
+
+exports.selectTopics = () => {
+  return db.query(`SELECT * FROM topics;`).then((result) => {
+    return result.rows;
+  });
+};
+
+exports.selectArticleById = (id) => {
+  if (isNaN(id))
+    return Promise.reject({status: 400, msg: "400 - article ID must be a number!"});
+  
+  return db.query(`SELECT * FROM articles WHERE article_id = ${id};`).then((result) => {
```

Comment:
You should refactor this to use the pg query parameters (e.g. `$1`) and the array of values to avoid the potential for sql injection.

See section 4. of these notes: https://notes.northcoders.com/courses/js-back-end/node-postgres